### ISSUE TITLE
Minor issue

### ISSUE THREAD

**Issue by JamesKrolak** [BarleyFlour/WrathVisualAdjustmentsMod#111]
If I right click on the mod in UMM (the .exe, not the in game menu) and select Home Page for this mod, it takes me to a page that doesn't exist.  It goes to:  https://github.com/BarleyFlour/WrathVisualAdjustmentsMod/Releases.  Can you change that address to take someone to just https://github.com/BarleyFlour/WrathVisualAdjustmentsMod?

**Comment by mkocus** [comment]
No, it should be changed to:
https://github.com/BarleyFlour/WrathVisualAdjustmentsMod/releases

(lowercase releases)

**Comment by platinawolf** [comment]
> 
> 
> No, it should be changed to: https://github.com/BarleyFlour/WrathVisualAdjustmentsMod/releases
> 
> (lowercase releases)

Agreed! But it's one of those tiiny tiny things that probably is looow on prio list...
